src/filesystem/impls/appshell/AppshellFileSystem.js

```diff
@@ -489,6 +489,19 @@ define(function (require, exports, module) {
     }
     
     /**
+     * Set the permissions for a file or directory at the given path, with the
+     * specified mode in numeric format (ie 0777)
+     * @param {string} path
+     * @param {number} mode
+     * @param {function(string)=} callback
+     */
+    function chmod(path, mode, callback) {
+        appshell.fs.chmod(path, mode, function (err) {
+            callback(_mapError(err));
+        });
+    }
```

Comment:
This is unused, right?  It's not accessible from anywhere else in Brackets (since FileSystem doesn't expose the fs impl object), so I think we should remove it.